manifests/virtual.pp

```diff
@@ -21,33 +21,37 @@
 #   node "toto.example.com" {
 #
 #     include postfix
-#
-#     postfix::hash { "/etc/postfix/virtual":
+# #     postfix::hash { "/etc/postfix/virtual":
 #       ensure => present,
 #     }
 #     postfix::config { "virtual_alias_maps":
 #       value => "hash:/etc/postfix/virtual"
 #     }
 #     postfix::virtual { "user@example.com":
 #       ensure      => present,
-#       destination => "root",
+#       destination => ['root', 'postmaster'],
 #     }
 #   }
 #
 define postfix::virtual (
-  String                    $destination,
-  Stdlib::Absolutepath      $file='/etc/postfix/virtual',
-  Enum['present', 'absent'] $ensure='present'
+  Variant[String, Array[String]] $destination,
+  Stdlib::Absolutepath           $file='/etc/postfix/virtual',
+  Enum['present', 'absent']      $ensure='present'
 ) {
   include ::postfix::augeas
 
+  $dest_sets = [$destination].flatten.map |$i, $d| {
+    $idx = $i+1
+    "set \$entry/destination[${idx}] '$d'"
```

Comment:
```suggestion
    "set \$entry/destination[${idx}] '${d}'"
```

Should fix CI

Reply:
Ah sorry, I saw your suggestion after I fixed it :smile: